.github/workflows/build-public-ami.yml

```diff
@@ -26,7 +30,7 @@ jobs:
       - name: Install aws cli
         run: |
           sudo apt update
-          sudo apt-get -y install packer python3-boto3
+          sudo apt-get -y install packer=${PACKER_VERSION} python3-boto3=${PYTHON3_BOTO3_VERSION}
```

Comment:
comment for line 50:

remove capital F
echo "SKIP_CREATE_AMI=false" >> $GITHUB_ENV